components/director/internal/repo/update.go

```diff
@@ -94,6 +96,7 @@ func (u *universalUpdater) unsafeUpdateSingle(ctx context.Context, dbEntity inte
 	if err = persistence.MapSQLError(err, u.resourceType, "while updating single entity"); err != nil {
 		return err
 	}
+	log.Debugf("Executing query: %s", stmtBuilder.String())
```

Comment:
i havent looked at the exact query logged but in general parameters (whatever goes in the place of ? placeholders) should not be logged (normally queries are logged without the parameters/arguments filled in). Or at the very least we should make sure we don't log passwords , keys and other sensitive information.

Reply:
In the actual logs the placeholders are not replaced with their actual values so there is no leak.